rll/enable-managed-login.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
+
+# ---
+# RightScript Name: RL10 Linux Enable Managed Login
+# Description: |
+#   Install RightScale NSS plugin, and update PAM and SSH configuration to
+#   allow SSH connectivity to RightScale accounts.
+# Inputs:
+#   MANAGED_LOGIN:
+#     Input Type: single
+#     Category: RightScale
+#     Description: To enable or disable managed login.  Default is 'enable'.
+#     Required: false
+#     Advanced: true
+#     Default: text:enable
+#     Possible Values:
+#       - text:enable
+#       - text:disable
+# Attachments:
+#   - rs-ssh-keys.sh
+#   - libnss_rightscale.tgz
+# ...
+
+set -e
+
+# Verify prerequisites before making changes
+if cut --delimiter=# --fields=1 /etc/ssh/sshd_config | grep --quiet AuthorizedKeysCommand; then
```

Comment:
I don't see how this will ignore the case where it is configured to use our script.